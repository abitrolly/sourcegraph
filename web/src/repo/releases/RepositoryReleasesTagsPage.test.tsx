import React from 'react'
import * as H from 'history'
import { RepositoryReleasesTagsPage } from './RepositoryReleasesTagsPage'
import { of } from 'rxjs'
import { shallow } from 'enzyme'

describe('RepositoryReleasesTagsPage', () => {
    const history = H.createMemoryHistory()
    test('renders', () =>
        expect(
            shallow(
                <RepositoryReleasesTagsPage
                    history={history}
                    location={history.location}
                    repo={{ id: '123' }}
                    queryGitReferences={() =>
                        of({
                            totalCount: 0,
                            nodes: [],
                            __typename: 'GitRefConnection',
                            pageInfo: { __typename: 'PageInfo', endCursor: '', hasNextPage: false },
                        })
                    }
                />
            )
        ).toMatchSnapshot())
})
