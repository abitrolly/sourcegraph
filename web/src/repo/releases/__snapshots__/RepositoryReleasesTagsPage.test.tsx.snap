// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepositoryReleasesTagsPage renders 1`] = `
<div
  className="repository-releases-page"
>
  <PageTitle
    title="Tags"
  />
  <FilteredConnection
    autoFocus={true}
    className="my-3"
    defaultFirst={20}
    history="[History]"
    listClassName="list-group list-group-flush"
    location="[Location path=/]"
    nodeComponent={[Function]}
    noun="tag"
    pluralNoun="tags"
    queryConnection={[Function]}
    useURLQuery={true}
  />
</div>
`;
